### PR TITLE
Added send email functionality to client and sample project 

### PR DESCRIPTION
**Pull request by avinashbr123**
- Add send email functionality to a client 
- Created send an email sample project

### CODE REVIEW

File: src/Kmd.Logic.Email.Client/ServicesMessages/AttachmentDetails.cs [hunk 1]
```diff
@@ -0,0 +1,14 @@
+﻿using System;
+
+namespace Kmd.Logic.Email.Client.ServicesMessages
+{
+    public class AttachmentDetails
```
Comment (nikhilcr939) [inline src/Kmd.Logic.Email.Client/ServicesMessages/AttachmentDetails.cs:5]: Since we have only AttachmentId property, can it be rename to AttachmentValue.

**Review by avin101 — APPROVED** [review]

File: src/Kmd.Logic.Email.Client/EmailClient.cs [hunk 2]
```diff
@@ -74,6 +76,54 @@ public async Task<MSExchangeConfigurationResponse> CreateMSExchangeConfiguration
...
+                sendEmailRequestDetails.CallbackUrl);
+
+            var configurationDetailsResponse = await client.SendEmailWithHttpMessagesAsync(
+                 this.options.SubscriptionId,
+                 request).ConfigureAwait(false);
+
+            switch (configurationDetailsResponse?.Response?.StatusCode)
+            {
+                case System.Net.HttpStatusCode.OK:
+                    return this.EmailResponce((SendEmailResponse)configurationDetailsResponse.Body);
```
Comment (smahapatra86) [inline src/Kmd.Logic.Email.Client/EmailClient.cs:119]: Should be `EmailResponse`. Not sure where `EmailResponce` is defined.

File: src/Kmd.Logic.Email.Client/EmailClient.cs [hunk 3]
```diff
@@ -89,6 +139,19 @@ public void Dispose()
...
 
+        private static IList<EmailAddress> MapEmailRecipients(IList<EmailAddressDetails> emails)
+        {
+            var lstEmail = new List<EmailAddress>();
+            emails?.ToList().ForEach(x => lstEmail.Add(new EmailAddress(x.Email)));
+            return lstEmail;
+            //return emails.Select(x => new EmailAddress(x.Email)).ToList();
+        }
+
+        private SendEmailResponseDetails EmailResponce(SendEmailResponse body)
```
Comment (smahapatra86) [inline src/Kmd.Logic.Email.Client/EmailClient.cs]: Should be `EmailResponse`

File: src/Kmd.Logic.Email.Client/ServicesMessages/EmailAddressDetails.cs [hunk 4]
```diff
@@ -0,0 +1,12 @@
+﻿namespace Kmd.Logic.Email.Client.ServicesMessages
+{
+    public class EmailAddressDetails
```
Comment (smahapatra86) [inline src/Kmd.Logic.Email.Client/ServicesMessages/EmailAddressDetails.cs:3]: If we can distinguish between `EmailAddress` and this class using some other naming convention that would be good.

File: src/Kmd.Logic.Email.Client/EmailClient.cs (hunk 3 above)
Comment (avinashbr123) [reply src/Kmd.Logic.Email.Client/EmailClient.cs]: Changed

File: src/Kmd.Logic.Email.Client/ServicesMessages/EmailAddressDetails.cs (hunk 4 above)
Comment (avinashbr123) [reply src/Kmd.Logic.Email.Client/ServicesMessages/EmailAddressDetails.cs:3]: As of new will keep it `EmailAddressDetails` only will change it later

**Review by smahapatra86 — APPROVED** [review]

**Pull request merged**